schema.sql

```diff
@@ -7,3 +7,12 @@ CREATE TABLE animals (
 INT NOT NULL, neutered BOOL NOT NULL,weight_kg DECIMAL NOT NULL);
 
  ALTER TABLE Animals ADD COLUMN SPECIES TEXT ;
+
+CREATE TABLE owners (id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, full_name TEXT NOT NULL,age INT NOT NULL);
+CREATE TABLE species (id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, name TEXT NOT NULL);
+ALTER TABLE Animals DROP COLUMN SPECIES;
+ALTER TABLE Animals ADD COLUMN id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY;
```

Comment:
- [ ] KIndly delete this statement. You already have an `ID` column on the `Animals` table and it is the primary key. When I run this file, I get this error:

![image](https://user-images.githubusercontent.com/77106962/207694461-18cb4c8b-471c-46b6-ad52-e9c7ead1d451.png)

#### Hint: If you want to auto-generate the `ID` anytime a row is created in the table, include the keyword `SERIAL`  for the id column. 👍

Reply:
Delete second animal table